src/components/CreateVmWizard/steps/Storage.js

```diff
@@ -105,6 +107,9 @@ class Storage extends React.Component {
     this.isEditingMode = this.isEditingMode.bind(this)
 
     this.state = {
+      editingErrors: {
```

Comment:
`editing` is an object that includes the data of the edited/created disk, and `creating`  just holds the id of the created disk in case of editing its value is false, so the answer is no, hope I explained it well :)

Creating disk:
![creating](https://user-images.githubusercontent.com/64131213/104180510-e7817700-5415-11eb-9945-eddae415de92.png)
Editing disk:
![editing](https://user-images.githubusercontent.com/64131213/104180935-9faf1f80-5416-11eb-83d3-02aa5657ccfe.png)